core/src/main/scala/zio/config/string.scala

```diff
@@ -0,0 +1,114 @@
+package zio.config
+
+import java.io.File
+import java.net.{ URI, URL }
+import java.time.{ Instant, LocalDate, LocalDateTime, LocalTime }
+import java.util.UUID
+
+import scala.concurrent.duration.Duration
+
+object string extends ReadFunctions with ConfigDocsFunctions {
```

Comment:
I'd suggest to name it not after `K` and `V` types (implementation details), but after its role like `default` or `common`.
And `string.string` looks a little bit misleading.

Reply:
`string` implies a `key` (at any level) is always `string` and the value associated with it will also be `string`.  But I like the name `common`. `default` in `config` can be misleading thinking we are dealing with `default values